Sources/SearchResults.swift

```diff
@@ -407,14 +433,29 @@ private func swift2Objc(_ matchLevel: MatchLevel_?) -> MatchLevel {
         guard let hits = content["hits"] as? [[String: Any]] else {
             throw InvalidJSONError(description: "Expecting attribute `hits` of type array of objects")
         }
-        self.hits = hits
+    
+        self.allHits = previousHits + hits
+        self.latestHits = hits
```

Comment:
Why this breaking change `hits -> latestHits`?

Reply:
These changes are not supposed to be in this PR, my bad for the incorrect way of rebasing. Refer to #19 as to why this change was made

Reply:
I get why we introduce `allHits`, but I don't get why we remove `hits`.